modules/adheseBidAdapter.js

```diff
@@ -166,4 +169,8 @@ function getAdDetails(ad) {
   return { creativeId: creativeId, dealId: dealId };
 }
 
+function base64urlEncode(s) {
+  return btoa(s).replace(/\+/g, '-').replace(/\//g, '_').replace(/\=+$/, '');
```

Comment:
Regarding the regex in the last `replace` call, you do not need to escape the `=` in `RegEx` (unless it's following a `?`).

Line 173 should be changed to the following:
`btoa('').replace(/\+/g, '-').replace(/\//g, '_').replace(/=+$/, '')`